controllers/articles.controller.js

```diff
@@ -34,4 +35,19 @@ const getArticlesCommentsById = (req, res, next) => {
     .catch((err) => next(err));
 };
 
-module.exports = { getArticles, getArticlesById, getArticlesCommentsById };
+const postComments = (req, res, next) => {
+  const { article_id } = req.params;
+
+  insertComments(req.body, article_id)
+    .then((comments) => {
+      res.status(201).send({ comments });
```

Comment:
You should only send back the newly posted comment here and it should be an object.